highfidelity/hifi
# Audio source sync

Description:
- Logic refactoring: there was several places/methods where audio was changed. Now it will use one model with audio devices and just update UI according to the model


Test plan
- change audio source in any audio menu
- make sure other menus reflects changes
- make sure HMD audio switches to/from HMD on HMD activate/deactivate
